python-sdk/docs/configurations.rst

```diff
@@ -27,11 +27,82 @@ We can also configure the default schema specific to the database type (example:
 
 .. code:: python
 
-   AIRFLOW__ASTRO_SDK__POSTGRES_DEFAULT_SCHEMA = "tmp"
+   AIRFLOW__ASTRO_SDK__POSTGRES_DEFAULT_SCHEMA = "postgres_tmp"
```

Comment:
@sunank200 we should also say we can modify all databases by one config `AIRFLOW__ASTRO_SDK__SQL_SCHEMA` and also the precedence in case `AIRFLOW__ASTRO_SDK__BIGQUERY_DEFAULT_SCHEMA` and `AIRFLOW__ASTRO_SDK__SQL_SCHEMA` is defined.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/861/commits/68f2bf333711b3c05666e56fb61073fe3853cb84)